.github/workflows/job_build_windows.yml

```diff
@@ -163,7 +164,7 @@ jobs:
         run: echo SSL_CERT_FILE=$(python3 -m certifi) >> $env:GITHUB_ENV
 
       - name: CMake configure
-        run: cmake -S ${{ env.OPENVINO_REPO }} -B ${{ env.BUILD_DIR }} ${{ inputs.cmake-options }}
+        run: cmake -S ${{ env.OPENVINO_REPO }} -B ${{ env.BUILD_DIR }} ${{ inputs.cmake-options }} -DCMAKE_CXX_FLAGS="/Zi" -DCMAKE_C_FLAGS="/Zi" -DCMAKE_SHARED_LINKER_FLAGS="/DEBUG" -DCMAKE_MODULE_LINKER_FLAGS="/DEBUG"
```

Comment:
Shouldn't we set those flags in the cmake project instead manually set for all jobs in CI?

Reply:
no, by default cmake does not generate PDB files for Release builds and by default users don't need them

For those, who want them, they are expected to set such flags externally. I have reflected it in dev documentation